yoti_test.go

```diff
@@ -628,6 +628,28 @@ func TestProfile_GetAttribute_String(t *testing.T) {
 	}
 }
 
+func TestEmptyStringIsAllowed(t *testing.T) {
+	attributeValueString := ""
+	attributeValue := []byte(attributeValueString)
+
+	var attr = &yotiprotoattr.Attribute{
+		Name:        attrConstGender,
+		Value:       attributeValue,
+		ContentType: yotiprotoattr.ContentType_STRING,
+		Anchors:     []*yotiprotoattr.Anchor{},
+	}
+
+	profile := createProfileWithSingleAttribute(attr)
+	att := profile.Gender()
+
+	if !cmp.Equal(att.Value(), attributeValueString) {
```

Comment:
Could we make use of https://godoc.org/gotest.tools/assert? Perhaps for another ticket/PR

Reply:
I've added SDK-865 for that 👍